pkg/resources/email_notification_integration_acceptance_test.go

```diff
@@ -12,11 +12,12 @@ import (
 )
 
 func TestAcc_EmailNotificationIntegration(t *testing.T) {
-	emailIntegrationName := strings.ToUpper(acctest.RandStringFromCharSet(10, acctest.CharSetAlpha))
-	if _, ok := os.LookupEnv("SKIP_EMAIL_INTEGRATION_TESTS"); ok {
+	env := os.Getenv("EMAIL_INTEGRATION_TESTS")
```

Comment:
Why we're switching to other env variable to check if we should skip ?

Reply:
reverted